tests/unit/api.py

```diff
@@ -1475,4 +1499,4 @@ def test_node_power_cycle(self, db):
         api.project_create('anvil-nextgen')
         api.node_register('node-99', 'ipmihost', 'root', 'tapeworm')
         api.project_connect_node('anvil-nextgen', 'node-99')
-        api.node_power_cycle('node-99')
+        api.node_power_cycle('node-99')
```

Comment:
I think that little arrow in the github diff is indicating that there isn't a final newline. Some tools break when that happens.